src/main/java/stsjorbsmod/patches/NlothsGiftPatch.java

```diff
@@ -0,0 +1,81 @@
+package stsjorbsmod.patches;
+
+import com.evacipated.cardcrawl.modthespire.lib.*;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.events.shrines.Nloth;
+import com.megacrit.cardcrawl.relics.AbstractRelic;
+import javassist.CannotCompileException;
+import javassist.CtBehavior;
+import javassist.expr.ExprEditor;
+import javassist.expr.FieldAccess;
+import stsjorbsmod.relics.FragileMindRelic;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+
+public class NlothsGiftPatch {
+
+    /**
+     * Removes FragileMindRelic if found in the iterator
+     *
+     * @param iter
+     */
+    public static void removeFragileMindRelic(Iterator<AbstractRelic> iter) {
+        while (iter.hasNext()) {
+            if (FragileMindRelic.ID.equals(iter.next().relicId)) {
+                iter.remove();
+            }
+        }
+    }
+
+    /**
+     * This is called only by edited expressions in the main game. See the following SpirePatch.
+     *
+     * @param playerRelics
+     * @return
+     */
+    public static List<AbstractRelic> clonePlayerRelicsWithoutFragileMind(List<AbstractRelic> playerRelics) {
+        List<AbstractRelic> relics = new ArrayList<>(playerRelics);
+        removeFragileMindRelic(relics.iterator());
+        return relics;
+    }
+
+    @SpirePatch(clz = AbstractDungeon.class, method = "getShrine")
+    public static class AbstractDungeon_getShrine_NlothCheck {
+        public static ExprEditor Instrument() {
+            return new ExprEditor() {
+                @Override
+                public void edit(FieldAccess fieldAccess) throws CannotCompileException {
```

Comment:
For testing it, I think a good option would would be to add a second temporary path to getShrine that uses `JorbsMod.logger` to print out the `tmp` list after it's generated.

Reply:
```
02:07:32.293 INFO stsjorbsmod.JorbsMod> [Match and Keep!, Wheel of Change, Golden Shrine, Transmorgrifier, Upgrade Shrine, Accursed Blacksmith, Designer, Duplicator, FaceTrader, Fountain of Cleansing, Lab, NoteForYourself, The Joust, The Woman in Blue]
```
output from that line with only the starting relics. N'loth is not there.